src/routes/Primary.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
!comment

Reply:
that is not a comment, instead is a type of special syntax that informs `eslint` to disable a default behaviour.